build-parent/pom.xml

```diff
@@ -68,6 +68,7 @@
         <version.plexus.util>3.5.1</version.plexus.util>
         <version.jackson.mapper.asl>1.9.3</version.jackson.mapper.asl>
         <version.exec.maven.plugin>3.4.1</version.exec.maven.plugin>
+        <version.commons.math>3.6.1</version.commons.math>
```

Comment:
A: We should pull this up to the root POM with the other `apache-commons-*` dependencies

Reply:
Well previously the commons was hardcoded in downstream projects like this:
```
        <dependency>
            <groupId>org.apache.commons</groupId>
            <artifactId>commons-math3</artifactId>
            <version>3.2</version>
        </dependency>
```
So rather than modify the entire migration logic to account for a to delete the version in this one case, figure it's fine to just use our new project `${version.commons.math}`

Reply:
Logic actually wouldn't be too much more complicated, so could go either way on this. I lean towards adding redundant version `<version>${version.commons.math}</version>` so that it's more obvious to the user where the version is being set.

Reply:
Went ahead and made the change to just remove the old version. Cleans up the baseline as well since we can use the managed dependency instead